akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -253,10 +255,7 @@ object Flow {
    *
    * '''Cancels when''' downstream cancels
    */
-  @Deprecated
-  @deprecated(
-    "Use lazyInitAsync instead. (lazyInitAsync returns a flow with a more useful materialized value.)",
-    "2.5.12")
+  @deprecated("Use 'Flow.lazyCompletionStageFlow' in combination with 'Flow.prefixAndTail(1)' instead", "2.6.0")
```

Comment:
is this Flow.prefixAndTail obvious? Maybe document in lazyCompletionStageFlow when and how that can be used?

Reply:
No, not obvious, I think we should add that as a sample since people have asked that now and then, but maybe that will be a fixme until there is a sample at all for it.

Reply:
Sample added